dnf-behave-tests/features/protected-running-kernel.feature

```diff
@@ -0,0 +1,60 @@
+Feature: Protected running kernel
+
+
+@bz1698145
+@no_installroot
+Scenario: Running kernel is protected
+  Given I use repository "repoquery-unsatisfied"
+   When I execute dnf with args "install dnf-ci-kernel"
+   Then the exit code is 0
+    And DNF Transaction is following
+        | Action        | Package                             |
+        | install       | dnf-ci-kernel-0:1.0-1.x86_64        |
+        | install-dep   | dnf-ci-systemd-0:1.0-1.x86_64       |
```

Comment:
Same here, you can do a single
```gherkin
Given I successfully execute dnf with args "install dnf-ci-kernel"
```

And put the shared `Given` steps into `Background`.

Reply:
Done.